packages/calico/extra/dcos-calico-bird.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
We will be always running an overlay network (vxlan/ipinip), right? Do we then still need to run bird on DC/OS?